alcs-frontend/src/app/features/application/decision/decision-v2/decision-component/nfup/nfup.component.ts

```diff
@@ -8,4 +8,9 @@ import { DecisionComponentDto } from '../../../../../../services/application/dec
 })
 export class NfupComponent {
   @Input() component!: DecisionComponentDto;
+  @Output() saveAlrArea = new EventEmitter<string | null>();
+
+  onSaveAlrArea($event: string | null) {
+    this.saveAlrArea.emit($event);
+  }
```

Comment:
I wonder if it will clean up component if you move this to the base component and nest all other decision components from it?

Reply:
Used <ng-content> to clean it up, dropped the MR by 60 lines :)